src/test/java/com/wikia/webdriver/testcases/discussions/EditingPostTests.java

```diff
@@ -0,0 +1,150 @@
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.remote.operations.DiscussionsOperations;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.DiscussionsConstants;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.PostEntity;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostDetailsPage;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostsListPage;
+import org.testng.annotations.Test;
+
+@Execute(onWikia = MercuryWikis.DISCUSSIONS_AUTO)
+@Test(groups = "discussions-editing-post")
+public class EditingPostTests extends NewTestTemplate {
+
+  private static final String USER_MOBILE_TEST_GROUP = "discussions-userMobileEditingPost";
+
+  private static final String USER_DESKTOP_TEST_GROUP = "discussions-userDesktopEditingPost";
+
+  private static final String DISCUSSIONS_ADMINISTRATOR_MOBILE_TEST_GROUP = "discussions-discussionsAdministratorMobileEditingPost";
+
+  private static final String DISCUSSIONS_ADMINISTRATOR_DESKTOP_TEST_GROUP = "discussions-discussionsAdministratorDesktopEditingPost";
+
+  private static final String EDITED_BY_PREFIX = "(edited by ";
+
+  private static final String EDITED_BY_ADMINISTRATORS = EDITED_BY_PREFIX + "administrators)";
+
+  private static final String EDITED_BY_QA_STAFF = EDITED_BY_PREFIX + "QATestsStaff)";
+
+  private static final String SHOULD_HAVE_EDITED_BY_SECTION_MESSAGE = "Post should have edited by section below post content.";
+
+  private static final String SHOULD_NOT_HAVE_EDITED_BY_SECTION_MESSAGE = "Post should not have information about who edited the post on posts list page.";
+
+  private static final String EDITED_BY_ADMINISTRATORS_MESSAGE = "Post should have information that it was edited by administrators.";
+
+  private static final String SAME_PERSON_MESSAGE = "If author and editor are the same person, edited by section should not be visible.";
+
+  private static final String SHOULD_HAVE_DETAILED_INFORMATION_MESSAGE = "Post should have detailed information by who it was edited.";
+
+  // User on mobile
+
+  @Test(groups = USER_MOBILE_TEST_GROUP)
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userOnMobileCanSeeThatPostWasEditedByAdministratorOnPostDetailsPage() {
+    final PostEntity.Data data = updatePostAsStaffRemotely(createPostAsUserRemotely());
+
+    final PostEntity post = new PostDetailsPage().open(data.getId()).getPost().findNewestPost();
```

Comment:
There's a possibility that - if this test group is executed in parallel with another - the post created above will not be the newest on the list. So the assertions below would fail. Do you think we could open post detail page instead? Or find this particular post by first getting to a page with posts by this user?